README.md

```diff
@@ -24,88 +22,117 @@ Via CDN:
     <script src="https://secure.aadcdn.microsoftonline-p.com/lib/<version>/js/msal.js"></script>
     <script src="https://secure.aadcdn.microsoftonline-p.com/lib/<version>/js/msal.min.js"></script>
 
- Note that msal.js is built for ES5, therefore enabling support for Internet Explorer 11. If you want to target Internet Explorer, you'll need to add a reference to promises polyfill.
+Internet Explorer does not have native `Promise` support, and so you will need to include a polyfill for promises such as `bluebird`.
 
     <!-- IE support: add promises polyfill before msal.js  -->
     <script src="https://cdnjs.cloudflare.com/ajax/libs/bluebird/3.3.4/bluebird.min.js" class="pre"></script>
 
-See here for more details on [supported browsers](https://github.com/AzureAD/microsoft-authentication-library-for-js/wiki/FAQs#q4-what-browsers-is-msaljs-supported-on).
+See here for more details on [supported browsers and known compatability issues](https://github.com/AzureAD/microsoft-authentication-library-for-js/wiki/FAQs#q4-what-browsers-is-msaljs-supported-on).
+
+## Roadmap and What To Expect From This Library
+Msal support on Javascript is a collection of libraries. msal-core or just simply msal, is the framework agnostic we library. Once our 1.X+ is stabilized, we are going to bring our msal-angular library to 1.x parity.  We are currently no planning on supporting msal-angularjs for 1.x, because the feedback we have heard is that community has moved on, however we will provide some code patterns of how you can use msal-core in an angularjs app. After our current libraries are up to standards, we will begin balancing new feature requests, with new platforms such as `react`, server side `node.js`, `electron`, etc...
+
+Our goal is communicate extremely well with the community and to take their opinions into account. We would like to get to a monthly minor release schedule, with patches comming as often as needed.  The level of communication, planning, and granularity we want to get to will be a work in progress.
+
+Please check our [roadmap](https://github.com/AzureAD/microsoft-authentication-library-for-js/wiki#roadmap) to see what we are workign on and what we are tracking next.
+
+## Oauth and the Implict Flow
```

Comment:
OAuth 2.0 to be explicit (since there was OAuth 1.0)

Reply:
"Working" instead of "workign" - typo

Reply:
updating good catch